### PR TITLE
Add Medarot 9: Kabuto Ver.

### PR DESCRIPTION
**Pull request by YukariMindGap**
Adds Medarot 9: Kabuto Ver..

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by YukariMindGap** [comment]
Here's the better boxart.
boxart
And here's the wiki:
https://github.com/citra-emu/citra-games-wiki/wiki/Medarot-9:-Kabuto-Ver.

**Comment by YukariMindGap** [comment]
I have no idea how to change boxart

**Comment by Leo626** [comment]
@YukariMindGap You make the changes in your [forked repository](https://github.com/YukariMindGap/citra-games-wiki/tree/pr-builder-medarot-9-kabuto-ver-47/games/medarot-9-kabuto-ver).

File: games/medarot-9-kabuto-ver/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Medarot 9: Kabuto Ver."
+description = "Medarot 9: Kabuto Version is the ninth installment of the Medarot series; an RPG series about battling each other with customizable robots. It was never released outside Japan."
```
Comment (Leo626) [inline games/medarot-9-kabuto-ver/game.dat]: Use this description instead.

> Medarot 9 is a role-playing video game developed by Delta Arts and published by Rocket Company, and the ninth installment in the Medarot series. Like previous games, it was released in Kabuto and Kuwagata versions.

**Pull request merged**